spec/presenters/tree_builder_ops_rbac_features_spec.rb

```diff
@@ -0,0 +1,63 @@
+describe TreeBuilderOpsRbacFeatures do
+  let(:features) do
+    %w(
+      all_vm_rules
+      instance
+      instance_view
+      instance_show_list
+      instance_control
+      instance_scan
+    )
+  end
+
+  let(:role) do
+    FactoryGirl.create(:miq_user_role, :id => 100_000_002, :features => features)
+  end
+
+  let(:tree) do
+    TreeBuilderOpsRbacFeatures.new(
+      "features_tree",
+      "features",
+      {:trees => {}},
+      true,
+      :role     => role,
+      :editable => false
+    )
+  end
+
+  let(:main_keys) do
+    bs_tree.first["nodes"].map { |n| n['key'] }
+  end
+
+  describe 'bs_tree' do
+    subject(:bs_tree) { JSON.parse(tree.locals_for_render[:bs_tree]) }
+
+    it 'builds the bs_tree' do
+      t = bs_tree.first
+
+      expect(t['key']).to match(/all_vm_rules/)
+      expect(t['title']).to be_nil
+      expect(t['tooltip']).to be_nil
+      expect(t['checkable']).to eq(false)
```

Comment:
This:
* `be_nil`
* `be_nil`
* `eq(false)`


Feels like:
![707](https://cloud.githubusercontent.com/assets/649130/24403654/1ef9f58c-13be-11e7-9fa9-a2a195492256.jpg)

Use `be_falsey` or `be false` instead :wink: